client/src/components/Meetup/Meetup.js

```diff
@@ -1,8 +1,689 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
 import PropTypes from "prop-types";
+import {
+  Avatar,
+  Box,
+  Button,
+  ButtonGroup,
+  Card,
+  CardContent,
+  CardHeader,
+  Chip,
+  CircularProgress,
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogContentText,
+  Grid,
+  IconButton,
+  Tooltip,
+  Typography,
+} from "@material-ui/core";
+import Link from "@material-ui/core/Link";
+import Paper from "@material-ui/core/Paper";
+import CheckIcon from "@material-ui/icons/Check";
+import ClearIcon from "@material-ui/icons/Clear";
+import EditIcon from "@material-ui/icons/Edit";
+import PersonAddDisabledIcon from "@material-ui/icons/PersonAddDisabled";
+import PersonAddIcon from "@material-ui/icons/PersonAdd";
+import { makeStyles } from "@material-ui/core/styles";
+import { Link as RouterLink } from "react-router-dom";
+import * as fetcher from "../../lib/fetch";
 
+const useStyles = makeStyles((theme) => ({
```

Comment:
I think quite a few of these classes are only used once. I typically prefer to inline the styles where they're actually used via `styles={{}}` for such classes.